code/go/0chain.net/smartcontract/dbs/event/transaction.go

```diff
@@ -51,3 +51,16 @@ func (edb *EventDb) GetTransactionByBlockHash(blockHash string, offset, limit in
 	res := edb.Store.Get().Model(Transaction{}).Where(Transaction{BlockHash: blockHash}).Offset(offset).Limit(limit).Scan(&tr)
 	return tr, res.Error
 }
+
+// GetTransactionByBlockNumbers finds the transaction record between two block numbers
+func (edb *EventDb) GetTransactionByBlockNumbers(blockStart, blockEnd int, offset, limit int) ([]Transaction, error) {
+	tr := []Transaction{}
+	res := edb.Store.Get().
+		Model(Transaction{}).
+		Joins("INNER JOIN  blocks on blocks.round >= ? AND blocks.round <= ? AND blocks.hash = transactions.block_hash", blockStart, blockEnd).
```

Comment:
There's an extra space between the 'JOIN  blocks'

Reply:
Great eye 🙏🏻 ! Fixed!!